modules/decline/src/Smithy4sCli.scala

```diff
@@ -111,24 +103,49 @@ class Smithy4sCli[Alg[_[_, _, _, _, _]], Op[_, _, _, _, _]](
         ).mapN { (input, entrypoint) =>
           val printers = entrypoint.printerApi
           val printer = printers.printer(endpoint)
-          printer.printInput(input) *>
+          val FO = printer.printInput(input) *>
             service
-              .asTransformation[GenLift[IO]#λ](entrypoint.interpreter)(
+              .asTransformation[GenLift[F]#λ](entrypoint.interpreter)(
                 endpoint.wrap(input)
               )
-              .flatMap(printer.printOutput)
-              .as(ExitCode.Success)
-              .recoverWith {
-                case e if endpoint.errorable.flatMap(_.liftError(e)).nonEmpty =>
-                  printer
-                    .printError(e)
-                    .as(ExitCode.Error)
-              }
+
+          FO.flatMap(printer.printOutput)
+            .onError {
+              case e if endpoint.errorable.flatMap(_.liftError(e)).nonEmpty =>
+                printer
+                  .printError(e)
+            }
         }
       }
   }
 
-  def main: Opts[IO[ExitCode]] = service.endpoints
+  private def opts: Opts[F[Unit]] = service.endpoints
     .foldMapK(endpointSubcommand(_))
 
+  def command: Command[F[Unit]] = {
+    Command(
+      toKebabCase(service.id.name),
+      header = service.hints
+        .get[Documentation]
+        .map(_.value)
+        .getOrElse(s"Command line interface for ${service.id.show}"),
+      helpFlag = true
+    )(opts)
+  }
+}
+
+object Smithy4sCli {
+  def Smithy4sSimpleStandaloneCli[Alg[_[_, _, _, _, _]], Op[_, _, _, _, _], F[
```

Comment:
Maybe let's rename this to `def standalone` - all the other words (except simple) appear in the context